### ISSUE TITLE
Lookin at rulesets and ruleset editors again

### ISSUE THREAD

**Issue by maxrchung** [maxrchung/S2VX#68]
Project on hold until we decide to continue again with rulesets

Working in an osu-ruleset may be much easier than dealing with osu-framework directly

This task will see the capabilities of rulesets and if we can easily port what we have now to rulesets

This task is dependent on #58. The ruleset must use the same osu-framework package that matches with S2VXStory.

**Comment by maxrchung** [comment]
Closed as we are doing our own game implementation
